### PR TITLE
UI test to delete user

### PR DESCRIPTION
**Pull request by paurakhsharma**
This is a UI test to check if the users gets deleted when an admin deletes them, and can no longer access the account.

### CODE REVIEW

**Comment by paurakhsharma** [comment]
cc @phil-davis Please review this test.

File: tests/acceptance/features/webUIManageUsersGroups/deleteUsers.feature [hunk 1]
```diff
@@ -0,0 +1,22 @@
+@webUI @insulated @disablePreviews
+Feature: manage users
```
Comment (phil-davis) [inline tests/acceptance/features/webUIManageUsersGroups/deleteUsers.feature]: feature is delete users

**Comment by phil-davis** [comment]
@paurakhsharma please make these same tests in ``core`` ``stable10``

**Comment by phil-davis** [comment]
Backport core `stable10` https://github.com/owncloud/core/pull/31930

**Pull request merged**